web/tests/test_scan.py

```
@@ -8,12 +9,15 @@
 import yaml
 from celery.utils.log import get_task_logger
 from reconPoint.settings import DEBUG
-from reconPoint.tasks import (fetch_url, http_crawl, port_scan, subdomain_discovery,
+from reconPoint.tasks import (dir_file_fuzz, fetch_url, http_crawl, initiate_scan,
+                           osint, port_scan, subdomain_discovery,
                            vulnerability_scan)
 from startScan.models import *
```

Ruff

`from startScan.models import *` used; unable to detect undefined names

(F403)